Test/TestProject/tests4.dfy

```diff
@@ -0,0 +1,200 @@
+// RUN: %dafny /compile:3 /coverage:- /spillTargetCode:1 /compileTarget:cs BranchCoverage2.cs "%s" > "%t"
```

Comment:
As discussed offline, this repo doesn't use Lit so this won't actually be run. We should figure out a way to validate the passthrough works though, similar to what we added for the verification params: https://github.com/dafny-lang/dafny.msbuild/blob/master/.github/workflows/msbuild.yml#L61